src/Security/Authentication/Negotiate/src/NegotiateExtensions.cs

```diff
@@ -17,15 +17,25 @@ namespace Microsoft.Extensions.DependencyInjection
     public static class NegotiateExtensions
     {
         /// <summary>
-        /// Adds Negotiate authentication.
+        /// Configures the <see cref="AuthenticationBuilder"/> to use Negotiate (also known as Windows, Kerberos, or NTLM) authentication
+        /// using the default scheme. The default scheme is specified by <see cref="NegotiateDefaults.AuthenticationScheme"/>.
+        /// <para>
+        /// Windows Authentication can be used if the web app is runs on a network using Active Directory domain identities or
+        /// Windows accounts to identify users. It requires support from the operating system to authenticate users.
+        /// </para>
```

Comment:
This actually supports general cross platform Kerberos, though we only tested it against AD.
```suggestion
```

Reply:
I used the text from the doc article here: https://docs.microsoft.com/en-us/aspnet/core/security/authentication/windowsauth?view=aspnetcore-3.1&tabs=visual-studio. Can we write a brief description of the auth that can be re-used there?

Reply:
That doc and description cover all of our windows auth implementations. This package is the only one that extends beyond windows auth to general kerberos.

How about "This authentication handler supports both Windows and Linux servers."?

Reply:
```suggestion
        /// <para>
        /// This authentication handler supports Kerberos on Windows and Linux servers.
        /// </para>
```

Reply:
How about that?

Reply:
👍